bin/rl/rl.cpp

```diff
@@ -4167,8 +4170,8 @@ ProcessConfig
 
       // Walk the condition nodes looking for applicability.
 
-      ConditionNodeList * conditionNodeList = NULL;
-      ConditionNodeList * conditionNodeLast = NULL;
+      conditionNodeList = NULL;
+      conditionNodeLast = NULL;
```

Comment:
I see why they have to be broken up. I personally think that variables should be explicitly initialized when they are declared, so the code as-is seems fine. 

There are sometimes performance reasons that we want to avoid explicit initialization. This wouldn't apply in this case, but for example, declaring a pointer and then passing a pointer to that pointer to a function as an outparam -- the compiler wouldn't elide an explicit initialization because it doesn't know about the semantics of that value when passed in to the function.

I just want to be sure that using `/permissive-` wouldn't prevent us from making that decision.